homeassistant/components/yeelight/__init__.py

```diff
@@ -324,7 +338,20 @@ def _update_properties(self):
 
         return self._available
 
+    def _get_capabilities(self):
+        """Read new capabilities from the device."""
```

Comment:
```suggestion
        """Request device capabilities."""
```

This docstring could include the details mentioned above, if really wanted.